static/less/btax/widget-table.less

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
No.  These are the only changes to css required by B-Tax (B-Tax also uses taxbrain.css).  The styles in `widget-table.less` relate to the styling with the deprecation table of check-all row at the top and rows below.